x/axelarnet/types/genesis.go

```diff
@@ -32,6 +37,7 @@ func DefaultGenesisState() *GenesisState {
 			AddrPrefix: "axelar",
 		}},
 		TransferQueue: utils.QueueState{},
+		IBCTransfers:  nil,
```

Comment:
```suggestion
		IBCTransfers:  nil,
		SeqIDMapping: nil,
```

Reply:
Is `nil` enough, or do you need to initialize them?

Reply:
added the field. yes it's enough, default is empty